airbyte-integrations/connectors/destination-iceberg/src/main/java/io/airbyte/integrations/destination/iceberg/config/catalog/IcebergCatalogConfigFactory.java

```diff
@@ -77,7 +77,7 @@ private static IcebergCatalogConfig genIcebergCatalogConfig(@NotNull JsonNode ca
       case HADOOP -> new HadoopCatalogConfig(catalogConfigJson);
       case JDBC -> new JdbcCatalogConfig(catalogConfigJson);
       case REST -> new RESTCatalogConfig(catalogConfigJson);
-      default -> throw new RuntimeException("Unexpected catalog config: " + catalogTypeStr);
```

Comment:
Please keep default.

Reply:
I think compiler will never reach for default case.  So I removed , please share your thoughts.

Reply:
@nk1506 I agree with @marcosmarxm, the default case of throwing should stay there